mixin.scss

```diff
@@ -56,7 +56,7 @@ $fonts-path: '../fonts' !default;
 		}
 		$src: append(
 			$src,
-			url('#{$fonts-path}/#{$font-family}/#{nth($font-file, 1)}.#{$type}')
+			url('#{$fonts-path}/#{$family-folder}/#{nth($font-file, 1)}.#{$type}')
```

Comment:
Not a bad idea but `$font-family-folder` should default to `$font-family` so that you don't **have** to specify the folder name to make it work. This will also allow it to be backwards compatible.

Reply:
In the documentation it says: "any spaces in the font name will get converted to dashes in the url" but this is not working, because $family-folder is not used, only set.

Reply:
Ah right, yeah. I haven't touched this project for a while. Thanks for the fix.

I'll merge and publish when I get the chance after I've done a bit of testing.